app/src/components/ClinicSelectionList.tsx

```diff
@@ -0,0 +1,107 @@
+import { Trans } from 'next-i18next'
+import {
+  ChangeEvent,
+  Dispatch,
+  MouseEvent,
+  ReactElement,
+  SetStateAction,
+} from 'react'
+
+import Button from '@components/Button'
+import ButtonLink, { ButtonLinkProps } from '@components/ButtonLink'
+import ClinicInfo from '@components/ClinicInfo'
+import Required from '@components/Required'
+
+import { Clinic } from '@src/types'
+
+export type ClinicSelectionListProps = {
+  filteredClinics: (Clinic | undefined)[]
+  expandList: boolean
+  setExpandList: Dispatch<SetStateAction<boolean>> | (() => unknown)
+  selectedClinic: Clinic | undefined
+  handleSelection: (e: ChangeEvent<HTMLInputElement>) => void
+  disabled: boolean
+  actionButton: ButtonLinkProps
+}
+
+const ClinicSelectionList = (props: ClinicSelectionListProps): ReactElement => {
```

Comment:
Is it customary to include the "ReactElement" type or could that be inferred? Just curious - haven't seen before I think! 

(it doesn't seem like a big value add since you'd usually infer that to be the return type for a react component!)

Reply:
Good call. I'll try dropping it.